manual/src/refman/extensions/effects.etex

```diff
@@ -759,6 +689,16 @@ computation. Given that the continuation captured in the shallow handler do not
 include the handler, there is only ever one handler installed in the dynamic
 scope of the computation "comp".
 
+Note that unlike deep handlers with syntax support, explicit type annotations
+are necessary for the shallow handler. We must use a locally abstract type
+"(type b)" in the effect handler ("effc") and explicitly type annotate the
+effect argument "eff" and the continuation "k" in each of the effect cases.
+Another point to note is that the catch-all effect case “| _ -> None” is
+necessary. This case may be intuitively read as “forward the unhandled effects
+to the outer handler”. The standard library module \stdmoduleref{Effect} also
+provides a non-syntactic version of deep handlers, where similar annotations
+are necessary.
+
```

Comment:
Is it worth putting in a short code example here of the type annotations, now that the previous uses of locally abstract types in the effect handlers are no longer doing that?

Reply:
Originally, I wanted to introduce a subsection before shallow handlers for "deep handlers sans syntax". But I felt the annotations were sufficiently covered by the shallow handler example above. Given that we have the syntax support for deep handlers, and the type annotations for non-syntactic deep handlers don't bring in anything new other than the tedium of writing down type annotations, I decided against it.